From ed8560719c6246182a77aca14c31ac736ff4d08a Mon Sep 17 00:00:00 2001
From: si-gui <sunguoshuai@huawei.com>
Date: Mon, 31 May 2021 14:44:03 +0800
Subject: [PATCH] fakeroot drop tartest
This patch comes from opensuse
(https://build.opensuse.org/package/view_file/openSUSE:Factory/fakeroot/fakeroot-drop-tartest.patch)
Skip tar test: the test is unstable and keeps on randomly failing.

---
 test/Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/Makefile.am b/test/Makefile.am
index 5ea4d72..93ce76e 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -9,7 +9,6 @@ TESTS =						\
   t.no_ld_preload				\
   t.no_ld_preload_link				\
   t.option					\
-  t.tar						\
   t.touchinstall				\
   t.truereturn \
   t.xattr
-- 
2.30.0

